NotesBackEnd/src/Notes.Api/App_Start/DependencyInjectionConfig.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Web.Http;
+using Notes.Api.Services.Dependency;
+using Notes.Api.Services.Services;
+using Notes.Contracts.Dependency;
+using Notes.Dependency.Containers;
+
+namespace Notes.Api
+{
+    internal static class DependencyInjectionConfig
+    {
+        internal static void Register(HttpConfiguration config)
+        {
+            IDependencyContainer container = new DependencyContainer()
+                .RegisterDependency(new Repository.Dependency.RepositoryTypesRegistration())
+                .RegisterDependency(new ApiServicesTypesRegistration());
+
+            config.DependencyResolver = new ContainerResolver(container);
+        }
+
+        private static IDependencyContainer RegisterDependency(this IDependencyContainer container, IBootstrapper registrationClass)
+            => registrationClass.RegisterType(container);

```

Comment:
This kind of looks like you register `container` to `registrationClass`. Perhaps the name of `RegisterType` could be a bit better.